test/files/cppy_source/test_extended_xsort.cppy

```diff
@@ -89,7 +89,7 @@ namespace xt
         check_argpartition_equal(py_a, py_a19, xt::argpartition(py_a, 19), 19);
 
         // py_median = np.median(a)
-        EXPECT_EQ(py_median, xt::median(py_a));
+        EXPECT_EQ(static_cast<long>(py_median), xt::median(py_a));
```

Comment:
I don't really mind, but just to know. Why is the cast needed here?

Reply:
Ditto